.github/workflows/check.yaml

```diff
@@ -28,4 +28,12 @@ jobs:
         fi
         echo -e "Found problematic uppercase section links:\n$found"
         exit 1
-    - run: yarn build
\ No newline at end of file
+    - run: yarn build
+    - name: Publish to Cloudflare Pages
+      uses: cloudflare/pages-action@1
+      with:
```

Comment:
```suggestion
      if: '! github.event.pull_request.draft'
      with:
```

Reply:
@mpetrunic do you think we also can add
```
     permissions:
        contents: read
        deployments: write
```

Section on the `steps` level so the `GITHUB_TOKEN` can be generated with enough permissions to update the repo deployments?

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs